tests/test_zipkin.py

```diff
@@ -2,12 +2,26 @@
 import gc
 import logging
 import tracemalloc
+from typing import Dict
 
 import pytest
 import aiozipkin as az
 from yarl import URL
 
 
+async def _retry_zipkin_client(url: URL, client: 'aiohttp.ClientSession',
+                               retries: int = 5) -> Dict:
+    tries = 0
+    while tries < retries:
+        asyncio.sleep(5)
```

Comment:
Probably good idea to add sleep time to the function arguments, as we have for `retries`, also lets make it lower say 1, just to speed up tests a bit.